jquery/exercises/domaccess.html

```diff
@@ -32,11 +31,10 @@
 <pre id="solution1" style="display:none;">
 $("img");
 $(".logo");
-$(".media");
-$(".tagline")
-$("a.green");
-$(".blue .topic");
-$("#donate a");
+$(".press-logos");
+$(".opener h1");
+$("header .navigation li:contains('donate')");
```

Comment:
Hmm. We don't cover contains(). Is there any other thing that'd be a tag plus a class, like 'a.green' was?